client/src/network/websocket_events/StatusEffects.js

```diff
@@ -59,21 +59,29 @@ export default () => {
 
     eventResponses.curse_set = (data) => {
```

Comment:
I think these entity lookups can now be cached like
`const entity = dungeonz.gameScene.dynamics[data];`
and then replace
`dungeonz.gameScene.dynamics[data]` with just `entity`, to make the lines not so long.